Update to Apex status - clarity on status (#3119)
forcedotcom/salesforcedx-vscode: docs/_articles/ja/user-guide/perf-enhancements.md

1. Library created and completed.
2. Available to try it in VS Code - We provide a VS Code setting so you can opt in and try it out. 
3. On by default in VS Code - The VS code setting default is defaulted to True to enable the library. If you previously opted in and then opted out, you'll have to manually opt back in to use the feature.  We'll change the default after the commands have been available and used for months.
4. CLI updated to use the library.  The VS Code setting is then retired.  Both the CLI and VS Code extensions are using the library.

|                         | Library complete	| Available to try in VS Code	| On by default in VS Code	| CLI Updated  |
|-------------------------|:-----------------:|:---------------------------:|:-------------------------:|:------------:|
| Apex Library	          |        ✔️          |               ✔️             |             ✔️             |       ✔️        |
| Deploy Retrieve Library |        ✔️          |               ✔️             |             ✔️              |              |
	

## Setup

> There is a VS Code setting for each library.  Access them by selecting Select **File** > **Preferences** > **Settings** (Windows or Linux) or **Code** > **Preferences** > **Settings** (macOS).  Setting to false means that VS Code will call the CLI to execute those commands. 

Reviewer: Make the same change here as above? Each library has a corresponding VS Code setting. Access them by selecting Select **File** > **Preferences** > **Settings** (Windows or Linux) or **Code** > **Preferences** > **Settings** (macOS). Setting to false means that VS Code uses the CLI to execute those commands.

Reviewer: I went ahead and suggested below to make it easier for you.

Reviewer 2: Thanks Emily! I added in the edit.

> There is a VS Code setting for each library.  Access them by selecting Select **File** > **Preferences** > **Settings** (Windows or Linux) or **Code** > **Preferences** > **Settings** (macOS).  Setting to false means that VS Code will call the CLI to execute those commands. 

Reviewer: ```suggestion
Each library has a corresponding VS Code setting. Access them by selecting Select **File** > **Preferences** > **Settings** (Windows or Linux) or **Code** > **Preferences** > **Settings** (macOS). Setting to false means that VS Code uses the CLI to execute those commands. 
```
